ui/src/wallet.ts

```diff
@@ -96,28 +96,30 @@ export function build(): Wallet {
   }
 
   async function disconnect() {
-    await walletConnect.killSession();
-    // We need to reinitialize `WalletConnect` until this issue is fixed:
-    // https://github.com/WalletConnect/walletconnect-monorepo/pull/370
-    walletConnect = new WalletConnect({
-      bridge: "https://bridge.walletconnect.org",
-      qrcodeModal: qrCodeModal,
-    });
-    signer.walletConnect = walletConnect;
-    stateStore.set({ status: Status.NotConnected });
+    try {
+      await walletConnect.killSession();
+    } catch {
+      // When the user disconnects wallet-side, calling `killSession` app-side trows an error
+      // because the wallet has already closed its socket. Therefore, we simply ignore it.
+    } finally {
```

Comment:
I usually try to reserve `finally` for cleanup code. To ignore errors it’s more idiomatic to use

~~~javascript
// When the user disconnects wallet-side, calling `killSession` app-side trows an error
// because the wallet has already closed its socket. Therefore, we simply ignore it.
await walletConnect.killSession().catch(() => {})
~~~

Reply:
Thanks for the input. Last time I used `try-catch` was 5 years ago in Java, and I don't remember it being possible to do such thing so I appreciate you providing cleaner ways :ok_hand:

Reply:
Done.